docker-entrypoint.sh

```diff
@@ -18,6 +18,8 @@ if [ ! -z "${DATABASE_URL}" ]; then
         bin/cake migrations seed -p BEdita/Core --seed AdminFromEnvSeed
     fi
 
+    chmod a+rwx /var/www/html/webroot/_files
```

Comment:
I think it'd be better to use `a+rwX`, since that sets the execution bit only on directories — and we don't want regular files uploaded by who-knows-who to be executable!

From man pages:
```
           x       The execute/search bits.
           X       The execute/search bits if the file is a directory or any of the execute/search bits are set in the original (unmodified) mode.  Operations with the perm symbol ``X''
                   are only meaningful in conjunction with the op symbol ``+'', and are ignored in all other cases.
```